jetty-util/src/main/java/org/eclipse/jetty/util/thread/Invocable.java

```diff
@@ -38,6 +38,35 @@
 
     static ThreadLocal<Boolean> __nonBlocking = new ThreadLocal<>();
 
+    public static Runnable from(InvocationType type, Runnable task)
+    {
+        class RunnableInvocable implements Runnable, Invocable
```

Comment:
This might be reusable if moved one level up?

Reply:
@gregw moved one level up (renamed to `Task`) and reused in other places.